Jenkinsfile

```diff
@@ -31,7 +31,7 @@ pipeline {
                 maven cmd: "clean install " +
                   "--settings deploy/single-project-over-http/settings.xml " +
                   "-Divy.engine.list.url=${params.engineListUrl} " +
-                  "-Divy.engine.version=[9.4.0,] " + 
+                  "-Divy.engine.version=[11.2,11.3\\) " + 

```

Comment:
any reason to not use the setting?

Reply:
I think we added it after 11.2? :thinking:  ... so I'd contribute the setting only to master: https://github.com/axonivy/project-build-examples/pull/171

Reply:
it's part of 11.2.1-SNAPSHOT
if you don't use the snapshot version here, then ok. why do you have \\ in the string?

Reply:
don't ask me about the escaping : didn't really understand it ... but it doesn't work without :sweat_smile: 

...anyway: we are using the latest snapshot: so I'll gladly use the new property instead of this